docs/src/main/asciidoc/_client-guide/authentication-client.adoc

```diff
@@ -292,6 +292,47 @@ The individual match elements are: -
 * <match-domain-name name="..."/>:: Matches where the protocol of the URI is 'domain' and the scheme specific part of the URI is the name specified within this match element.
  <match-abstract-type name="..." authority="..." />:: Matches the abstract type and/or authority against the values specified within this match element.
 
+=== Elytron client default SSL context security provider
+
+You can register the `org.wildfly.security.auth.client.ElytronClientDefaultSSLContextProvider` security provider. This java security provider loads Elytron client configuration and provides an initialized default SSL context that is available by calling `SSLContext.getDefault()`.
```

Comment:
I'd add a note here, to make clear that the provider needs to have high enough priority, WDYT?

Reply:
@fabiobrz I specified that the provider must be registered with high enough priority.